.github/workflows/enforce-license-compliance.yml

```diff
@@ -2,15 +2,19 @@ name: Enforce License Compliance
 
 on:
   push:
-    branches: [master, release/*]
+    branches:
+      - master
+      - release/**
+      - release-library/**
+
   pull_request:
     branches: [master]
 
 jobs:
   enforce-license-compliance:
     runs-on: ubuntu-latest
     steps:
-      - name: 'Enforce License Compliance'
+      - name: "Enforce License Compliance"
```

Comment:
I'm not sure we don't have competing linting rules in different repos, but I can't find a concrete counterexample right now.